connectivity/cellular/source/framework/RIL/RIL_CellularContext.cpp

```diff
@@ -0,0 +1,1145 @@
+/*
+ * Copyright (c) 2020, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "RIL_CellularContext.h"
+#include "RIL_CellularDevice.h"
+#include "RIL_CellularNetwork.h"
+#include "RILAdaptation.h"
+#include "LWIPStack.h"
+#include "CellularLog.h"
+#include "ScopedLock.h"
+
+#define ril_itoa(v, s, d)    sprintf(s, "%d", v)
```

Comment:
this doesn't make sense to me, what is the intention here? how do you think it should work?

Reply:
From a security perspective (or even just tracing some rare bug in the future) this looks pretty bad.